python/foglamp/plugins/south/cc2650async/cc2650async.py

```diff
@@ -0,0 +1,368 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Module for Sensortag CC2650 'async' type plugin """
+
+import copy
+import datetime
+import uuid
+import json
+import asyncio
+
+from foglamp.plugins.south.common.sensortag_cc2650 import *
```

Comment:
Generally, __all__ only comes into play when the from xxx import * variant of the import statement is used. This applies to packages as well as to modules.